app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,10 @@
   <string name="cellular_data_alert_dialog_title">Currently on Cellular Data</string>
   <string name="cellular_data_alert_dialog_description">Streaming audio may use a lot of cellular data.</string>
   <string name="cellular_data_alert_dialog_checkbox">Don\'t show this message again</string>
+  <string name="stop_exploration_dialog_title">Leave to Topic Page?</string>
+  <string name="stop_exploration_dialog_description">Your progress will be saved.</string>
+  <string name="stop_exploration_dialog_cancel_button">CANCEL</string>
```

Comment:
This and below should be capitalized via the theme, not here. Please make these normal cased.

Reply:
Done. Made all text in regular case.